frontend/src/views/database/mysql/conn/index.vue

```diff
@@ -80,16 +80,19 @@
                             :rules="Rules.paramComplexity"
                             prop="password"
                         >
-                            <el-input type="password" show-password clearable v-model="form.password">
-                                <template #suffix>
-                                    <CopyButton type="icon" :content="form.password" class="w-30" />
-                                </template>
-                                <template #append>
-                                    <el-button @click="random">
-                                        {{ $t('commons.button.random') }}
-                                    </el-button>
-                                </template>
-                            </el-input>
+                            <el-input
+                                style="width: calc(100% - 147px)"
+                                type="password"
+                                show-password
+                                clearable
+                                v-model="form.password"
+                            />
+                            <el-button-group>
+                                <CopyButton :content="form.password" />
+                                <el-button @click="random">
+                                    {{ $t('commons.button.random') }}
+                                </el-button>
+                            </el-button-group>
                         </el-form-item>
                     </div>
 
```

Comment:
The only change appears to be moving the CopyButton to an el-button-group component, which might improve the spacing between buttons in small screens. However, beyond styling adjustments, there are no irregularities or other concerns with this code snippet. If you have more specific questions on optimizing performance, accessibility, or best practices related to form elements, feel free to ask!